samples/cf-multi-tenant-application/README.md

```diff
@@ -0,0 +1,11 @@
+# SAP Cloud SDK for JavaScript Multi-Tenant Sample Application
+
+This code samples in this directory are a reference for the [multi-tenant application](https://sap.github.io/cloud-sdk/docs/js/tutorials/multi-tenant-application) tutorial.
+You need to make adjustments to the code before deployment described in the tutorial.
```

Comment:
The line is a bit confusing. Are the code adjustments described in the tutorial? Or you mean 'You need to make adjustments to the code in the tutorial before deployment.'

```suggestion
You need to make adjustments to the code before deployment.
```

Reply:
Maybe we can add a hint about which placeholders need to be replaced? Like we do in the cf-sample-app?